frontend/components/RewardsHistory/RewardsHistory.tsx

```diff
@@ -180,16 +181,24 @@ const ContractRewards = ({ checkpoints }: ContractRewardsProps) => {
   );
 };
 
+/**
+ * TODO: Refactor, only supports a single service for now
+ * */
 export const RewardsHistory = () => {
   const { contractCheckpoints, isError, isLoading, isFetching, refetch } =
     useRewardsHistory();
   const { goto } = usePageState();
   const { selectedService } = useServices();
-  const serviceId = selectedService?.service_config_id;
-  const chainId = selectedService?.home_chain_id;
+
+  const serviceConfigId = selectedService?.service_config_id;
+  const serviceNftTokenId =
```

Comment:
a bit afraid this renaming here and in other places can be confusing. it's called `serviceId` in contracts and other apps (weird it's called token on BE though, but maybe I'm missing something).
do you think it might add more confusion? cc @mohandast52

Reply:
It is confusing but need some way to differentiate between them as service config id and service id (from the nft) now both exist